src/main/java/com/urbanairship/connect/client/StreamConsumeTask.java

```diff
@@ -95,6 +97,7 @@ private void stream() throws ConnectionException {
         while (active.get()) {
 
             Optional<StartPosition> position = getPosition();
+            log.debug("stream position: " + position);
```

Comment:
I think it's good to be even more explicit for these purposes-- "Opening new stream connection at position"

Reply:
That is better, updated.